README.md

```diff
@@ -303,6 +303,16 @@ To enable inner join add this to your configuration.
 ```
 
 
+There is also an ORM Query Type for LeftJoin.  This join type is commonly used to fetch an empty right side of a relationship.
+
+Left Join is not included by default in the `zf-doctrine-querybuilder.global.php.dist`.
```

Comment:
I've removed the instructions to add the filter yourself and I've added both join filters to the .dist config file.  https://github.com/TomHAnderson/zf-doctrine-querybuilder/commit/f7d911985a0babc906dfa8d3e7ccc828a2f68c2f